doc/source/contribution/documentation/introduction.rst

```diff
@@ -15,43 +15,53 @@ Groonga uses Sphinx_ as documentation tool.
 
 Here are command lines to install Sphinx.
 
-Debian GNU/Linux, Ubuntu::
+Debian GNU/Linux, Ubuntu:
 
-  % sudo apt-get install -V -y python3-pip gettext
-  % sudo pip install -r doc/requirements.txt
-  % (cd doc && bundle install)
+.. code-block:: console
 
-CentOS, Fedora::
+   % sudo apt-get install -V -y python3-pip gettext
+   % sudo pip install -r doc/requirements.txt
+   % (cd doc && bundle install)
 
-  % sudo yum install -y python-pip gettext
-  % sudo pip install -r doc/requirements.txt
-  % (cd doc && bundle install)
+CentOS, Fedora:
```

Comment:
We should use AlmaLinux instead of CentOS in a separated PR.

Reply:
Sure. I didn't know that the CentOS users would use the AlmaLinux because of EOL.

Reply:
fix: https://github.com/groonga/groonga/pull/1723 I also fixed the name from OS X to macOS.